index.html

```diff
@@ -114,6 +114,79 @@ <h4 class="heading heading_level-4">Подушка</h4>
+                        </div>
+                        <div class="column column_s-2 column_m-2 column_l-4">
+                            <div class="features__item">
+                                <img class="features__icon" src="./images/support-icon.svg" alt="Круглосуточная служба поддержки"/>
+                                <p class="features__name">Круглосуточная служба поддержки</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <footer class="footer">
+            <div class="column-wrapper">
+                <div class="column-row">
+                    <div class ="footer__main">
+                        <div class="column column_s-2 column_m-3 column_l-4">
+                            <nav class="footer__menu">
+                                <ul class="footer__list">
+                                    <li><a class="footer__link footer__link_facebook" href="#" target="_blank"></a></li>
+                                    <li><a class="footer__link footer__link_twitter" href="#" target="_blank"></a></li>
+                                    <li><a class="footer__link footer__link_vk" href="#" target="_blank"></a></li>
+                                    <li><a class="footer__link footer__link_instagram" href="#" target="_blank"></a></li>
+                                </ul>
+                            </nav>
+                        </div>
+                        
+                    
+                        <div class="column column_s-2 column_m-3 column_l-4">
+                            <div class="footer__buttons-container">
+                                <a class="footer__button footer__button_app-store"></a>
+                                <a class="footer__button footer__button_google-play"></a>
+                            </div>
+                        </div>
+
+                        <div class="column column_s-2 column_m-3 column_l-4">
+                            <img class="footer__logo" src="./images/hh-footer-logo.svg" alt="HH logo"/>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="column-row">
+                    <div class="column column_s-2 column_m-6 column_l-12">
+                        <div class="footer__line"></div>
+                    </div>
+                </div>
+            
+                <div class="column-row">
+                    <div class="column column_s-2 column_m-6 column_l-12">
+                        <div class="footer__info">
+                            <p class ="footer__text">© 2022 Группа компаний HeadHunter</p>
+                            <p class ="footer__text">Сегодня на сайте 684&nbsp;978&nbsp;вакансий, 34&nbsp;789&nbsp;747&nbsp;резюме, 858&nbsp;923&nbsp;компании и за неделю 1&nbsp;211&nbsp;363&nbsp;приглашения</p>
+                        </div>
```

Comment:
класс из стилей удалил а здесь забыл, ну и этот враппер перестал иметь смысл - можно удалить
```suggestion
                        <p class ="footer__text">© 2022 Группа компаний HeadHunter</p>
                        <p class ="footer__text">Сегодня на сайте 684&nbsp;978&nbsp;вакансий, 34&nbsp;789&nbsp;747&nbsp;резюме, 858&nbsp;923&nbsp;компании и за неделю 1&nbsp;211&nbsp;363&nbsp;приглашения</p>
```